packages/create-medusa-app/src/utils/nextjs-utils.ts

```diff
@@ -0,0 +1,107 @@
+import inquirer from "inquirer"
+import promiseExec from "./promise-exec.js"
+import { FactBoxOptions, displayFactBox } from "./facts.js"
+import fs from "fs"
+import { customAlphabet, nanoid } from "nanoid"
+import { isAbortError } from "./create-abort-controller.js"
+import logMessage from "./log-message.js"
+import { EOL } from "os"
+
+const NEXTJS_REPO = "https://github.com/medusajs/nextjs-starter-medusa"
+
+export async function askForNextjsStarter(): Promise<boolean> {
+  const { installNextjs } = await inquirer.prompt([
+    {
+      type: "confirm",
+      name: "installNextjs",
+      message: `Would you like to install the Next.js storefront? You can also install it later`,
```

Comment:
suggestion: Would `create` or `add` make more sense than `install` here? 🤔

Reply:
Are we talking about both instances of `install`?